requirements/requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy
-ovos-plugin-manager~=0.0.19
-ovos-utils~=0.0.25
-streaming-stt-nemo==0.0.4a1
+ovos-plugin-manager~=0.0.19,>=0.0.21a6
+ovos-utils>=0.0.25
```

Comment:
`ovos-utils~=0.0.25`? Or, `ovos-utils~=0.0.25,>=0.0.27a7` (latest alpha release)

Reply:
got it.